docs/int/openapi.yaml

```diff
@@ -1148,6 +1354,33 @@ components:
         name:
           type: string
 
+    NamedReferenceResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uri
+        name:
+          type: string
+        value:
+          type: string
+        type:
+          $ref: '#/components/schemas/KeywordType'
+        framework:
+          type: string
+
+    NamedReferenceRequest:
```

Comment:
Rename to 'NamedReferenceUpdate' to match other create/update schemas
- [ ] rename schema